### PR TITLE
DOC: remove an extra colon

### PR DESCRIPTION
**Pull request by partev**
fix a typo by removing an extra colon

- [ ] closes #xxxx
- [ ] tests added / passed
- [ ] passes `black pandas`
- [ ] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

### CODE REVIEW

File: doc/source/user_guide/gotchas.rst [hunk 1]
```diff
@@ -321,7 +321,7 @@ Byte-ordering issues
 --------------------
 Occasionally you may have to deal with data that were created on a machine with
 a different byte order than the one on which you are running Python. A common
-symptom of this issue is an error like:::
+symptom of this issue is an error like::
```
Comment (MarcoGorelli) [inline doc/source/user_guide/gotchas.rst:324]: Thanks @partev  - aren't there two extra colons though?

```suggestion
symptom of this issue is an error like:
```

**Review by MarcoGorelli — CHANGES_REQUESTED** [review]

File: doc/source/user_guide/gotchas.rst (hunk 1 above)
Comment (jorisvandenbossche) [reply doc/source/user_guide/gotchas.rst:324]: @MarcoGorelli the two colons are a restructuredtext construct to ensure the following block is a code block

**Review by jorisvandenbossche — APPROVED** [review]

**Comment by jorisvandenbossche** [comment]
@partev thanks!

File: doc/source/user_guide/gotchas.rst (hunk 1 above)
Comment (MarcoGorelli) [reply doc/source/user_guide/gotchas.rst:324]: ah oops - thanks Joris!

**Pull request merged**